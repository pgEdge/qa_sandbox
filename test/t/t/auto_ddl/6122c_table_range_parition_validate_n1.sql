SELECT pg_sleep(1);--to ensure all objects are replicated

-- This file runs on n1 again to see all the table and their partitions have been dropped on n1 (as a result of drop statements)
-- being auto replicated via 6122b
SET ROLE appuser;

SET search_path TO s612, public;

-- none of these tables should exist.
\d+ sales_range
\d+ sales_range_2021
\d+ sales_range_2022
\d+ sales_range_2023
--spock.tables should be empty
SELECT * FROM get_table_repset_info('sales');

\d+ revenue_range
\d+ revenue_range_2021
\d+ revenue_range_2022
\d revenue_range
--spock.tables should be empty
SELECT * FROM get_table_repset_info('revenue');

\d+ orders_range
\d+ orders_range_2021
\d+ orders_range_2022
--spock.tables should be empty
SELECT * FROM get_table_repset_info('orders');

\d+ inventory_range
\d+ inventory_range_2021
\d+ inventory_range_default
\d+ inventory_standalone
--spock.tables should be empty
SELECT * FROM get_table_repset_info('inventory');

RESET ROLE;
--dropping the schema
DROP SCHEMA s612 CASCADE;