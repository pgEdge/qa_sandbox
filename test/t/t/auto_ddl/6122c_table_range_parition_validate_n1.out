SELECT pg_sleep(1);--to ensure all objects are replicated
 pg_sleep 
----------
 
(1 row)

-- This file runs on n1 again to see all the table and their partitions have been dropped on n1 (as a result of drop statements)
-- being auto replicated via 6122b
SET ROLE appuser;
SET
SET search_path TO s612, public;
SET
-- none of these tables should exist.
\d+ sales_range
Did not find any relation named "sales_range".
\d+ sales_range_2021
Did not find any relation named "sales_range_2021".
\d+ sales_range_2022
Did not find any relation named "sales_range_2022".
\d+ sales_range_2023
Did not find any relation named "sales_range_2023".
--spock.tables should be empty
SELECT * FROM get_table_repset_info('sales');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ revenue_range
Did not find any relation named "revenue_range".
\d+ revenue_range_2021
Did not find any relation named "revenue_range_2021".
\d+ revenue_range_2022
Did not find any relation named "revenue_range_2022".
\d revenue_range
Did not find any relation named "revenue_range".
--spock.tables should be empty
SELECT * FROM get_table_repset_info('revenue');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ orders_range
Did not find any relation named "orders_range".
\d+ orders_range_2021
Did not find any relation named "orders_range_2021".
\d+ orders_range_2022
Did not find any relation named "orders_range_2022".
--spock.tables should be empty
SELECT * FROM get_table_repset_info('orders');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

\d+ inventory_range
Did not find any relation named "inventory_range".
\d+ inventory_range_2021
Did not find any relation named "inventory_range_2021".
\d+ inventory_range_default
Did not find any relation named "inventory_range_default".
\d+ inventory_standalone
Did not find any relation named "inventory_standalone".
--spock.tables should be empty
SELECT * FROM get_table_repset_info('inventory');
 nspname | relname | set_name 
---------+---------+----------
(0 rows)

RESET ROLE;
RESET
--dropping the schema
DROP SCHEMA s612 CASCADE;
INFO:  DDL statement replicated.
DROP SCHEMA
